src/components/UserMenu/UserMenu.tsx

```diff
@@ -33,9 +46,52 @@ const UserMenu = ({ showPaymentsItem, small = false, onClick }: Props) => {
 
   return (
     <ul className={styles.menuItems}>
+      {accessModel === 'SVOD' && profilesEnabled && (
```

Comment:
What do you think of wrapping this in another component, e.g UserMenuProfiles, ProfilesMenu, that can still be under UserMenu dir.